civihr_employee_portal/src/View/CustomFieldsMappingFixer.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace Drupal\civihr_employee_portal\View;
+
+/**
+ * To match unknown table names and field names when importing views
+ * with custom fields and custom groups from CiviCRM.
+ * Replaces unknown fields names and unknown tables like:
+ * - exported [name_of_the_field]_[ID1] with existing [name_of_the_field]_[ID2]
+ * - exported [name_of_the_table]_[ID3] with existing [name_of_the_table]_[ID4]
+ * ... where ID1, ID2, ID3 & ID4 are different numbers created by different
+ * mechanisms thus producing different IDs.
+ */
+class CustomFieldsMappingFixer {
+
+  /**
+   * To fix mapping on Views import of CiviCRM custom fields
+   *
+   * @param view $view
+   *     the view that is being imported / recreated
+   */
+  static public function fixMapping($view) {
```

Comment:
Please always add the visibility modifier first. I.e. `public static`. This applies to all the methods on this class